api.go

```diff
@@ -204,25 +209,27 @@ func (c *Client) vagetbackendauth(s url.URL, cid string) error {
 	// Set Authorization Header
 	req.Header.Add("x-vchs-authorization", c.VAToken)
 
-	// Adding exponential backoff to retry
-	b := backoff.NewExponentialBackOff()
-	b.MaxElapsedTime = time.Duration(30 * time.Second)
```

Comment:
Exponential backoff should stay as there are timeout issues with the backend auth on subscription.